src/ui-test/tests/command.palette.test.ts

```diff
@@ -0,0 +1,101 @@
+import * as path from 'path';
+import {
+    ActivityBar,
+    after,
+    before,
+    BottomBarPanel,
+    EditorView,
+    InputBox,
+    SideBarView,
+    VSBrowser,
+    WebDriver,
+    Workbench,
+} from 'vscode-uitests-tooling';
+
+describe('JBang commands execution through command palette', function () {
+    this.timeout(240000);
+
+    //const CAMEL_RUN_DEBUG_ACTION_COMMAND = 'Run Camel Application with JBang and Debug';
+    const CAMEL_RUN_ACTION_COMMAND = 'Run Camel Application with JBang';
+    const CAMEL_ROUTE_YAML_WITH_SPACE = 'demo route.camel.yaml';
+
+    const TEST_ARRAY_RUN = [
+        "Routes startup",
+        "Hello Camel from yaml"];
+
+    let activityBar: ActivityBar;
+    let editorView: EditorView;
+    let driver: WebDriver;
+
+    before('Before setup', async function () {
+        driver = VSBrowser.instance.driver;
+    });
+
+    after('After cleanup', async function () {
+        await new EditorView().closeAllEditors();
+    });
+
+    beforeEach('Before each test', async function () {
+        activityBar = new ActivityBar();
+        await (await activityBar.getViewControl('Explorer')).openView();
```

Comment:
same here.. more details comment below

Reply:
Agreed, for now, will delete it.
But it will be useful when we will add Run&Debug to be confident that we are in the explorer section (tested)